vocabularies/Org.OData.Temporal.V1.xml

```diff
@@ -67,214 +67,192 @@
         <Parameter Name="timeslices" Type="Collection(Edm.EntityType)" Nullable="false">
           <Annotation Term="Core.Description" String="Time slices to modify" />
         </Parameter>
-        <Parameter Name="ObjectKey" Type="Edm.EntityType" Nullable="false">
-          <Annotation Term="Core.Description" String="Property values that identify a single temporal object" />
-          <Annotation Term="Core.LongDescription" String="Select-expand projection of the same entity type as binding parameter, containing all and only properties reference in the [`ObjectKey`](#ObjectKey) of that entity type." />
-        </Parameter>
-        <Parameter Name="From" Type="Edm.PrimitiveType" Nullable="false">
-          <Annotation Term="Core.Description" String="Lower boundary (inclusive) of application time. Date or DateTimeOffset value, or string with value `min`" />
-        </Parameter>
-        <Parameter Name="To" Type="Edm.PrimitiveType" Nullable="true">
-          <Annotation Term="Core.Description" String="Optional upper boundary (exclusive) of application time. Date or DateTimeOffset value, or string with value `max`" />
-        </Parameter>
-        <Parameter Name="Values" Type="Edm.EntityType" Nullable="false">
-          <Annotation Term="Core.Description" String="Property values to update" />
-          <Annotation Term="Core.LongDescription" String="Same entity type as binding parameter and return type, specify only values of non-key properties." />
+        <Parameter Name="newTimeslices" Type="Collection(Edm.EntityType)" Nullable="false">
+          <Annotation Term="Core.Description" String="New time slices whose property values are used to update the `timeslices` collection" />
+          <Annotation Term="Core.LongDescription">
+            <String>Same entity type as binding parameter.
+            The new time slices need not contain all properties, but at least the lower boundary of the period to change.
+            An absent object key property matches any key property value.
+            New time slices are processed in the order of the collection, which especially matters if some of the specified change periods overlap.</String>
+          </Annotation>
         </Parameter>
         <ReturnType Type="Collection(Edm.EntityType)" Nullable="false">
           <Annotation Term="Core.Description" String="Modified time slices" />
           <Annotation Term="Core.LongDescription" String="Same entity type as binding parameter." />
         </ReturnType>
       </Action>
 
-      <Action Name="SynchronizeObject" IsBound="true" EntitySetPath="timeslices">
+      <Action Name="Synchronize" IsBound="true" EntitySetPath="timeslices">
         <Annotation Term="Core.Description" String="Synchronizes time slices of a temporal object to match the provided full application timeline" />

```

Comment:
Unclear what this means. Is `timeslices` simply replaced with `timeline` (per object key)?

Reply:
The resulting history is semantically equivalent, i.e. the same values at the same point in time, but potentially with a different cut of time slices - we can't require services to merge adjacent and value-identical time slices.